db/migrate/20210222175117_add_sent_followup_to_signup.rb

```diff
@@ -0,0 +1,5 @@
+class AddSentFollowupToSignup < ActiveRecord::Migration[6.0]
+  def change
+    add_column :signups, :sent_followup, :boolean, default: false
```

Comment:
I think its nice to create things like this as a timestamp because it does that same indication of boolean value for us but also gives us some more interesting data that we can use to track like ... how long it took someone to go from being sent the followup to signing up, or how long it took us to send all the followups out, etc.